src/core/utils/chainProviders.ts

```diff
@@ -8,6 +8,23 @@ import { CustomMulticallJsonRpcProvider } from './CustomMulticallJsonRpcProvider
 import { logger } from './logger'
 import { MulticallQueue } from './multicall'
 
+const ChainMulticallSettings: Record<
+  Chain,
+  { flushTimeoutMs: number; maxBatchSize: number }
+> = Object.values(Chain).reduce(
+  (accumulator, chainId) => {
+    return {
+      [chainId]: {
+        flushTimeoutMs: 0.1,
+        // Allow a bigger batch size for mainnet
+        maxBatchSize: Chain.Ethereum ? 200 : 100,
```

Comment:
Making it configurable via env is possible, but we would need to pass some sort of json string with the values per chain.

In the meantime, I think that reducing the batch size for non-mainnet chains is a good policy.